etc/acl.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../../lib/internal/Magento/Framework/Acl/etc/acl.xsd">
```

Comment:
Should use the urn instead of the path `urn:magento:framework:Acl/etc/acl.xsd`

Reply:
Good call, updated :+1: